simulationTool/components/Ensemble/EnsembleDetails.vue

```diff
@@ -368,157 +370,132 @@ export default {
+                    border-right: 2px solid var(--bs-default);
                 }
 
-                div.status {
-                    display: inline-block;
-                    padding: .25rem .5rem;
-                    border-radius: .5rem;
-                    font-size: .875rem;
-                    font-weight: 500;
-                    color: white;
-
-                    &.accepted {
-                        background-color: var(--bs-info);
-                    }
-
-                    &.running {
-                        background-color: var(--bs-warning);
-                    }
-
-                    &.successful {
-                        background-color: var(--bs-success);
-                    }
-
-                    &.dismissed {
-                        background-color: var(--bs-secondary);
-                    }
-
-                    &.failed {
-                        background-color: var(--bs-danger);
-                    }
+                &.action-column {
+                    width: 40px;
+                    padding: 0;
                 }
             }
 
-            .details-header {
-                grid-area: header;
+            th {
+                white-space: nowrap;
             }
 
-            .models {
+            td > div {
+                white-space: normal;
+                display: -webkit-box;
+                line-clamp: 2;
+                -webkit-line-clamp: 2;
+                -webkit-box-orient: vertical;
                 overflow: hidden;
-                grid-area: models;
-                display: flex;
-                flex-direction: column;
-                flex-grow: 1;
-                overflow: auto;
-                min-height: 0;
+                text-overflow: ellipsis;
+                word-break: break-word;
             }
 
-            .jobs {
-                overflow: hidden;
-                grid-area: jobs;
-                min-height: 200px;
-            }
-
-            .toolbar {
-                grid-area: toolbar;
-                display: flex;
-                justify-content: flex-end;
-            }
+            .status {
+                display: inline-block;
+                padding: .25rem .5rem;
+                border-radius: .5rem;
+                font-size: .875rem;
+                font-weight: 500;
+                color: white;
 
-            .notes {
-                grid-area: notes;
-                overflow: hidden;
-                display: inherit;
-            }
-
-            .sharing {
-                grid-area: sharing;
-                overflow: hidden;
-                display: inherit;
+                &.accepted { background-color: var(--bs-info); }
+                &.running { background-color: var(--bs-warning); }
+                &.successful { background-color: var(--bs-success); }
+                &.dismissed { background-color: var(--bs-secondary); }
+                &.failed { background-color: var(--bs-danger); }
```

Comment:
I'm not sure if these are still needed as there is global config for these in the SimulationTool.vue.

Reply:
They are removed now